src/Interactive/EditorFeatures/Core/Extensibility/Interactive/InteractiveEvaluator.cs

```diff
@@ -192,8 +192,8 @@ private void ProcessStarting(InteractiveHostOptions options)
             var metadataService = _workspace.CurrentSolution.Services.MetadataService;
             ImmutableArray<string> referencePaths;
 
-            // reset configuration:
-            if (File.Exists(_responseFilePath))
+            // reset configuration (null initialization file indicates "noconfig"):
+            if (options.InitializationFile != null && File.Exists(_responseFilePath))
```

Comment:
I see...I would say:

- consider null checking options (```options?.InitializationFile != null```)
- consider using ```options.InitializationFile``` here (and Assert'ing that it's the same as ```_responseFilePath```)